src/components/shared/inputs/ImageUploader.tsx

```diff
@@ -102,10 +103,12 @@ export default function ImageUploader({
               textOverflow: 'ellipsis',
             }}
           >
-            Uploaded to:{' '}
-            <a href={url} target="_blank" rel="noopener noreferrer">
-              {url}
-            </a>
+            <Trans>
+              Uploaded to:{' '}
+              <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
should this URL be inside the <Trans/> component?

Reply:
Yep, same as above